docs/.vuepress/theme/styles/dark.styl

```diff
@@ -2,7 +2,7 @@
 .show-dark
   display none !important
 
-#wrapper.dark-mode
+.dark-mode
```

Comment:
Not sure if this is the change that caused it, but the dark highlight is now incorrect:
<img width="332" alt="Image 2020-02-28 at 12 57 00 PM" src="https://user-images.githubusercontent.com/8097623/75550454-dad87800-5a29-11ea-8e0d-8381f160e930.png">

Reply:
this is a recurring thing in the recent PRs when reducing specificity. resolved